src/sentry/quotas/redis.py

```diff
@@ -61,53 +63,56 @@ def get_quotas(self, project, key=None, keys=None):
 
         results = []
 
-        pquota = self.get_project_quota(project)
-        if pquota[0] is not None:
-            results.append(
-                QuotaConfig(
-                    id="p",
-                    scope=QuotaScope.PROJECT,
-                    scope_id=project.id,
-                    categories=DataCategory.error_categories(),
-                    limit=pquota[0],
-                    window=pquota[1],
-                    reason_code="project_quota",
+        with sentry_sdk.start_span(op="redis.get_quotas.get_project_quota"):
```

Comment:
Let's maybe add `project.id` as a span attribute, to quickly see the offending project.